### ISSUE TITLE
Is it possible to change the naming of geometries loaded from primitives?

### ISSUE THREAD

**Issue by ibozhilov** [mikedh/trimesh#1017]
I am using trimesh to load gltf files. Some of the meshes contains multiple primitives and as such I really need to distinguish the geometries generated from those primitives. Right now the name for the geometry is generated from the name of the mesh, underscore and then the index of the primitive:

```
# each primitive gets it's own Trimesh object
if len(m["primitives"]) > 1:
    name += "_{}".format(j)
```

Is it possible to be something like
```
# each primitive gets it's own Trimesh object
if len(m["primitives"]) > 1:
    name += "_primitive{}".format(j)
```
Thanks,
Ivan

**Comment by mikedh** [comment]
Good idea that would make sense! Or the other thing that might allow for more programmatic access is a field in `scene.metadata`, maybe something like `gltfPrimitives : {gltfMesh : [geometryName, geometryName1, ...]}`

**Comment by mikedh** [comment]
If I get a chance I'll try to do something like that, unfortunately maybe not near term, but PR's super welcome!

**Comment by ibozhilov** [comment]
I have thought about it second time and think that the thing with the name is not very smart, however I still need to be able to get it from the trimesh object, so I have create a PR #1185 where I create a new metadata field called is_from_primitive, which is set to True, when the trimesh object is created from GLTF primitive. Would appreciate if you take a look.
